collections/_priorities/platforms-00-integrate-accesskit-to-improve-accessibility.md

```diff
@@ -0,0 +1,19 @@
+---
+last_updated: "2024-11-30"
+category: platforms
+rank: 0
+created: "2024-11-30"
+stage:
+  # Must be dates (YYYY-MM-DD), empty if not there yet.
+  planning: ""
+  ongoing: ""
+  done: ""
+anchor: "integrate-accesskit-to-improve-accessibility"
+title: "Integrate AccessKit to improve accessibility"
+description: |
+  [AccessKit](https://github.com/AccessKit/accesskit) offers a cross-platform way to implement accessibility features for UI toolkits, such as Godot. It would offer us and game developers opportunities to make more accessible products, such as the Godot Editor itself or games made with Godot.
```

Comment:
```suggestion
  [AccessKit](https://github.com/AccessKit/accesskit) offers a cross-platform way to implement accessibility features for UI toolkits (screen reader support), such as Godot. It would offer us and game developers opportunities to make more accessible products, such as the Godot Editor itself or games made with Godot.
```

This should probably mention it is `screen reader support` specifically, or it might be confused with more game specific accessibility features (which is not related to).

Also, can be marked as `ongoing`.